workshop/4-Messaging/pinpoint-templates/abandoned-cart-email-template.html

```diff
@@ -1,27 +1,508 @@
-<!DOCTYPE html>
-    <html lang="en">
-    <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<!doctype html>
```

Comment:
Oh, how I love HTML emails. Ugh. These changes look good but I think the Pinpoint workshop notebook may need to be changed to not display the template source any longer because the template is much bigger now and may be overwhelming to expect the reader to absorb it all.

Also, what about the text version of the template? If we're not going to keep the text version is sync with the HTML template, we can consider dropping the text templates all together.

Reply:
Thats a good point. Are the text templates used/ providing any value for the demo / workshop modes? If not, I can drop them

Reply:
The text templates are not providing much value. Really just showing customers how they can specify both an HTML and text template for end customers using email clients that support/require one or the other. I say just drop the text templates. Be sure to update the Pinpoint notebook and pre-create Lambda.

Reply:
As discussed offline, I have updated the text template to reflect the HTML template.
I am keeping the text files so as to avoid making changes in the workshop as a part of this PR

Reply:
Updated the workshop description and replaced viewing the HTML templates with snippets as discussed offline